app/components/post-filter-type.js

```diff
@@ -0,0 +1,22 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  classNames: ['button-group', 'dropdown'],
+  classNameBindings: ['hidden:menu-hidden:menu-visible'],
+
+  hidden: true,
+
+  active: Ember.computed.not('hidden'),
+
+  actions: {
+    toggle: function() {
+      this.toggleProperty('hidden');
+    },
+    hide: function() {
+      this.set('hidden', true);
+    },
+    filterByType: function(type) {
+      this.sendAction('filterByType', type);
+    },
+  }
+});
```

Comment:
Maybe I'm missing a binding somewhere, but couldn't we just use the `active` property directly here? `toggle` would toggle it as always, `hide` would set it to false.